Automattic/vip-go-mu-plugins
# Support concatenation outside of VIP environments

## Description


If a local or non-VIP environment is correctly configured to run the concatenation plugin, it should be able to use the same logic as VIP's environments.

## Changelog Description


## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Set the `VIP_GO_ENABLE_HTTP_CONCAT` constant in `wp-config.php` or `vip-config.php` and confirm that the concatenation plugin respects the constant.
